### DISCUSSION TITLE
当鼠鬚管部署失敗时，去哪裏查看錯誤日誌？

### DISCUSSION THREAD

**Discussion by pdog18** [rime/home#1285] [Q&A] [answered]
<img width="382" alt="image" src="https://user-images.githubusercontent.com/17060376/235448899-8326a774-b177-4fa0-b82b-24f1d504e8ab.png">

出現這種通知以後，點擊 `show` 完全沒有反應。

**Comment by pdog18** [comment]
而我使用 termail 進入 `$TMPDIR` 后，只能發現这样的情況：
<img width="367" alt="image" src="https://user-images.githubusercontent.com/17060376/235449024-85ce192c-a251-40de-a24a-29114e255f0b.png">

**Reply by LEOYoon-Tsaw** [reply, marked as answer]
就是這三個文件，可以用vim打開
`vim $TMPDIR/rime.squirrel.ERROR`

**Reply by pdog18** [reply]
> 就是這三個文件，可以用vim打開 `vim $TMPDIR/rime.squirrel.ERROR`

打开以后发现是一个空的文件

**Reply by LEOYoon-Tsaw** [reply]
通知顯示的是看INFO，還是該看INFO

**Reply by pdog18** [reply]
> 通知顯示的是看INFO，還是該看INFO

我同时查看了 INFO、ERROR及WARING, 都是空文件..

**Reply by pdog18** [reply]
<img width="627" alt="Pasted Graphic 5" src="https://user-images.githubusercontent.com/17060376/235716445-210972a6-0482-4f62-8d3d-3082cafd1800.png">
<img width="639" alt="Pasted Graphic 6" src="https://user-images.githubusercontent.com/17060376/235716482-74e8667c-f262-443e-b7c2-8f1ddb7d27bc.png">
我故意制造了一个错误, 然后就是无法看到日志

**Comment by LEOYoon-Tsaw** [comment]
<img width="682" alt="image" src="https://user-images.githubusercontent.com/4469383/235722973-ee251d90-8ada-4c5a-91f6-e53e415d312f.png">
我的有

**Reply by pdog18** [reply]
那看来真的是一个 bug 了.
